ansible/environments.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Ensure aeternity network environments
+  hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Ensure integration environment stack
+      os_stack:
```

Comment:
I see this relies on OS_* env variables for auth.

Reply:
Yes, the README has separate section on [credentials setup](https://github.com/aeternity/infrastructure/pull/5/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17)